diff -up kcalc-19.04.1/org.kde.kcalc.desktop.omv~ kcalc-19.04.1/org.kde.kcalc.desktop
--- kcalc-19.04.1/org.kde.kcalc.desktop.omv~	2019-05-10 16:27:07.672798472 +0200
+++ kcalc-19.04.1/org.kde.kcalc.desktop	2019-05-10 16:27:28.344651921 +0200
@@ -149,5 +149,5 @@ GenericName[zh_TW]=科學計算機
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=Qt;KDE;Utility;Calculator;X-KDE-Utilities-Desktop;
+Categories=Qt;KDE;Utility;Calculator;X-KDE-Utilities-Desktop;Education;Science;Math;
 X-DBUS-ServiceName=org.kde.kcalc.desktop
